app/Console/Commands/PollTwitter.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Log;
+
+class PollTwitter extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'suggestive:poll-twitter';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Polls Twitter for any tweets with the configured hashtag.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $users = User::where('role', 'owner')
+                     ->whereNotNull('hashtag')
+                     ->get(['id', 'name', 'hashtag', 'last_tweet_id']);
+
+        if ($users) {
+            Log::info('Found owners to poll Twitter for', ['count' => $users->count()]);
+
+            $twitter = app('TwitterClient');
+
+            $users->each(function ($user) use ($twitter) {
+                Log::info('Polling Twitter', ['owner' => $user->name, 'hashtag' => $user->hashtag]);
```

Comment:
I suggested queue a job here. Your question was:

> What's the theory behind Job vs scheduled Command? What would the trigger be to the queue the job to execute? Or are you just suggesting that I queue the request and response part of the command so that it doesn't need to handle that part of it?

I would recommend something like this:

```php
$users->each(function ($user) {
    $this->dispatch(new PollTwitterForUsersHashtag($user));
}
```

The benefit of separating the logic here is that you A) have encapsulated a single user's poll logic in the same way this artisan command is encapsulating a single cron-ready action of "do all the polls for this site", and B) if the twitter pulls are slow or hitting rate limits or whatever, each individual user's pull is separate and queued, which means they're not holding up the whole stack... you can throw multiple queue listeners.. you can delete them after a certain number of failures... etc. 

Basically, queuing tasks like this gives you a much greater level of control and flexiblity.

In response to your question, I'm still 100% in favor of making this an artisan command, but I just like it to be a command that then queues jobs.